.gitignore

```diff
@@ -89,6 +89,6 @@ sw.*
 # content
 /static/*.ordered.*
 /static/*.menu.*
-routes.json
+#routes.json
```

Comment:
Hey @manuelschroederdev could you please revert this? I believe it's just a snag, right?